README.md
## Air-light demo content

This mu-plugin provides only the demo content for [Air-light](https://github.com/digitoimistodude/air-light) theme as presented in [airwptheme.com/demo](https://airwptheme.com/demo).

No further documentation needed.


CHANGELOG.md
### 2.0.8: 2022-08-25

* Present accessibility features alongside new external-link.js

### 2.0.7: 2022-08-23

* Fix image class for Air-light v9.0.8

### 2.0.6: 2022-06-09

* Fix breakpoints and paddings

### 2.0.5: 2022-06-09

* Reponsive fixes for demo site
* Fix nav button on in-between resolutions
* Disable air-light placeholder in demo page ("Build your website here! (Remove this bit in sass/layout/_site-header.scss)")
* Add max-width for demo blocks

### 2.0.4: 2022-05-26

* Fix padding of the centered block without image

### 2.0.3: 2022-05-26

* Fix paddings on demo templates other than home
* Add block name tags to single, blog and 404 views

### 2.0.2: 2022-05-26

* Fix placeholder for the upcoming Air-light v.9.0.1

### 2.0.1: 2022-05-25

* Fix typo in variable
* Fix button links

### 2.0.0: 2022-05-25

* Add phpcs.xml
* Completely rewrite demo blocks
* Completely rewrite new styles for demo site
* Hide air-light "Start building here" placeholder on demo
* Get ready for Air-light v9.0.0

### 1.0.6: 2022-05-12

* Remove link to non-existing page

### 1.0.5: 2022-04-21

* Fix z-index bug with cssnano on Air-light 8.3.7

### 1.0.4: 2022-02-08

* Fix mobile styles for demo content.

### 1.0.3: 2022-02-08

* Add demo styles for container-less and hero-less Air-light 8.3.1.

### 1.0.2: 2021-11-04

* Support for new link component in Air-light 8.1.0

### 1.0.1: 2021-09-24

* Rename gutenberg-content to article-content according to [Air-light 7.9.0](https://github.com/digitoimistodude/air-light/releases/tag/7.9.0)

### 0.0.9: 2021-09-15

* Validate HTML, fix exactly the following reports
* Warning: The "type" attribute for the "style" element is not needed and should be omitted
* Error: CSS: "fill": "var(--color-baby-blue)" is not a valid color 3 or 6 hexadecimals numbers

### 0.0.8: 2021-06-15

* Add examples of long list items

### 0.0.7: 2021-05-29

* Fix demo content heading margin

### 0.0.6: 2021-05-29

* Various mobile fixes

### 0.0.5: 2021-05-24

* Fix plane size

### 0.0.4: 2021-05-24

* Fix some style issues on demo page

### 0.0.3: 2021-05-24

* Fix the hook name

### 0.0.2: 2021-05-18

* Add dummy styles to admin as well

### 0.0.1: 2021-05-18

* Release plugin


block-demo-content.php
<?php
/**
 * Custom block
 *
 * @Author: Timi Wahalahti
 * @Date:   2021-05-17 13:34:52
 * @Last Modified by:   Roni Laukkarinen
 * @Last Modified time: 2022-08-25 14:19:42
 *
 * @package air-light
 */

?>
<section class="block block-hero-big-image">
  <div class="image-background" aria-hidden="true">
      <img loading="lazy" alt="A decorative picture about ferris wheel" src="https://blocks.airwptheme.com/media/bruno-abatti-Xzer0SqKPEE-unsplash.jpeg">

      <div class="container">
        <div class="content">
          <div class="content-wrapper">

            <h1 id="content" class="has-text-gradient has-blue-text-gradient">
              Air-light is a WordPress starter theme for developers
            </h1>

            <p>You are looking at thousands of hours of development work over a decade. Our aim is to build the greatest base for new WordPress websites out there. Remember, this is just a demo.</p>

            <p>This demo has custom blocks enabled, but air-light doesn't ship with them. You'll need to build your blocks yourself.</p>

            <p class="button-wrapper">
              <a class="button button-large" href="https://airwptheme.com">
                See all features
              </a>
            </ul>
          </div>
        </div>
      </div>

    </div>
<div class="block-name-tag" aria-hidden="true"><p>hero</p></div>
</section>

<section class="block block-content-columns block-accessibility image-left">
  <div class="container">

    <div class="col col-content">
      <div class="wrapper">
        <h2 class="has-text-gradient">Accessible by nature</h2>
        <p>Air-light is 100% accessible. It's a joy to browse with screen readers, keyboards and other assistive technology. Our navigation closes with esc even when hovering it with a mouse (try it)! We even have automated external link screen reader labels and visual arrow indicators for links. Our theme is constantly tested with assistive tools like NVDA.</p>
        <p class="button-wrapper">
          <a class="button no-external-link-indicator has-icon" href="https://github.com/digitoimistodude/air-light">
            <svg width="16" height="16" viewBox="0 0 1792 1792" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M1664 896q0 251-146.5 451.5T1139 1625q-27 5-39.5-7t-12.5-30v-211q0-97-52-142 57-6 102.5-18t94-39 81-66.5 53-105T1386 856q0-121-79-206 37-91-8-204-28-9-81 11t-92 44l-38 24q-93-26-192-26t-192 26q-16-11-42.5-27T578 459.5 492 446q-44 113-7 204-79 85-79 206 0 85 20.5 150t52.5 105 80.5 67 94 39 102.5 18q-40 36-49 103-21 10-45 15t-57 5-65.5-21.5T484 1274q-19-32-48.5-52t-49.5-24l-20-3q-21 0-29 4.5t-5 11.5 9 14 13 12l7 5q22 10 43.5 38t31.5 51l10 23q13 38 44 61.5t67 30 69.5 7 55.5-3.5l23-4q0 38 .5 89t.5 54q0 18-13 30t-40 7q-232-77-378.5-277.5T128 896q0-209 103-385.5T510.5 231 896 128t385.5 103T1561 510.5 1664 896z"></path></svg>
            Get started now
          </a>
        </p>

      </div>
    </div>

    <div class="col col-image" aria-hidden="true">
      <div class="wrapper">
        <p>
          <span class="pre-label">This regular external link</span>
          <a href="https://github.com/digitoimistodude/air-light">Air-light GitHub</a>
        </p>

        <p><span class="pre-label">Becomes</span></p>
        <pre class="code code-small">&lt;<span style="color:#e289bc;">p</span>&gt;
  &lt;<span style="color:#e289bc;">a</span> <span style="color:#7ccb90;">aria-label</span>="<span style="color:#c2b957">Air-light GitHub: External site</span>" <span style="color:#7ccb90;">href</span>=<span style="color:#c2b957">"https://github.com/digitoimistodude/air-light"</span>&gt;
    Air-light GitHub
    &lt;<span style="color:#e289bc;">svg</span>
      <span style="color:#7ccb90;">aria-hidden</span>=<span style="color:#c2b957">"true"</span>
      <span style="color:#7ccb90;">xmlns</span>=<span style="color:#c2b957">"http://www.w3.org/2000/svg"</span>
      <span style="color:#7ccb90;">width</span>=<span style="color:#c2b957">"12"</span>
      <span style="color:#7ccb90;">height</span>=<span style="color:#c2b957">"12"</span>
      <span style="color:#7ccb90;">viewBox</span>=<span style="color:#c2b957">"0 0 9 9"</span>&gt;
        &lt;<span style="color:#e289bc;">path</span> d=<span style="color:#c2b957">"M4.499 1.497h4v4m0-4l-7 7"</span>
          <span style="color:#7ccb90;">stroke</span>=<span style="color:#c2b957">"currentColor"</span>
          <span style="color:#7ccb90;">fill</span>=<span style="color:#c2b957">"none"</span>
          <span style="color:#7ccb90;">fill-rule</span>=<span style="color:#c2b957">"evenodd"</span>
          <span style="color:#7ccb90;">stroke-linecap</span>=<span style="color:#c2b957">"round"</span>
          <span style="color:#7ccb90;">stroke-linejoin</span>=<span style="color:#c2b957">"round"</span>&gt;
        &lt;/<span style="color:#e289bc;">path</span>&gt;
      &lt;/<span style="color:#e289bc;">svg</span>&gt;
  &lt;/<span style="color:#e289bc;">a</span>&gt;
&lt;/<span style="color:#e289bc;">p</span>&gt;</pre>
      </div>
    </div>

  </div>
<div class="block-name-tag" aria-hidden="true"><p>content-columns</p></div>
</section>

<section class="block block-demo-centered has-image">
  <div class="container">
    <h2 class="has-text-gradient">Gutenberg blocks make the website easy to update</h2>
    <p>Air-light has 100% Gutenberg support. Get magical with custom ACF + Gutenberg blocks.</p>
    <p class="button-wrapper">
      <a class="button no-external-link-indicator has-icon" href="https://docs.airwptheme.com/air-blocks/block-library">
        Get started with block library
      </a>
    </p>
  </div>

  <div class="image-background">
    <img loading="lazy" src="https://blocks.airwptheme.com/media/gutenberg.png" alt="Gutenberg-editor">
  </div>

<div class="block-name-tag" aria-hidden="true"><p>demo-centered</p></div>
</section>

<section class="block block-content-columns">
  <div class="container">

    <div class="col col-content">
      <div class="wrapper">
        <h2 class="has-text-gradient">Modular SCSS + CSS custom properties for everything</h2>
        <p>When it comes to Cascading Style Sheets, we've got you covered. Air-light is bloat-free! Just open _colors.scss and you are good to go. Did you know that Air-light production-ready CSS weights only 99KB without compression?</p>
        <p class="button-wrapper">
          <a class="button no-external-link-indicator has-icon" href="https://github.com/digitoimistodude/air-light">
            <svg width="16" height="16" viewBox="0 0 1792 1792" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M1664 896q0 251-146.5 451.5T1139 1625q-27 5-39.5-7t-12.5-30v-211q0-97-52-142 57-6 102.5-18t94-39 81-66.5 53-105T1386 856q0-121-79-206 37-91-8-204-28-9-81 11t-92 44l-38 24q-93-26-192-26t-192 26q-16-11-42.5-27T578 459.5 492 446q-44 113-7 204-79 85-79 206 0 85 20.5 150t52.5 105 80.5 67 94 39 102.5 18q-40 36-49 103-21 10-45 15t-57 5-65.5-21.5T484 1274q-19-32-48.5-52t-49.5-24l-20-3q-21 0-29 4.5t-5 11.5 9 14 13 12l7 5q22 10 43.5 38t31.5 51l10 23q13 38 44 61.5t67 30 69.5 7 55.5-3.5l23-4q0 38 .5 89t.5 54q0 18-13 30t-40 7q-232-77-378.5-277.5T128 896q0-209 103-385.5T510.5 231 896 128t385.5 103T1561 510.5 1664 896z"></path></svg>
            GitHub
          </a>
        </p>

      </div>
    </div>

    <div class="col col-image" aria-hidden="true">
      <div class="wrapper">
        <ul class="vars">
          <li>&ndash;<span class="spacer"></span>&ndash;<span class="spacer"></span>color-brand-twitter: <span class="color" style="background-color: #1da1f2"></span> #1da1f2</li>
          <li>&ndash;<span class="spacer"></span>&ndash;<span class="spacer"></span>color-pattens-blue: <span class="color" style="background-color: #eaf1f8;"></span> #eaf1f8;</li>
          <li>&ndash;<span class="spacer"></span>&ndash;<span class="spacer"></span>color-science-blue: <span class="color" style="background-color: #0049d1;"></span> #0049d1;</li>
          <li>&ndash;<span class="spacer"></span>&ndash;<span class="spacer"></span>color-scorpion: <span class="color" style="background-color: #5f5f5f;"></span> #5f5f5f;</li>
          <li>&ndash;<span class="spacer"></span>&ndash;<span class="spacer"></span>color-valhalla: <span class="color" style="background-color: #2a2d3e;"></span> #2a2d3e;</li>
        </ul>
      </div>
    </div>

  </div>
<div class="block-name-tag" aria-hidden="true"><p>content-columns</p></div>
</section>

<section class="block block-components block-content-columns image-left">
  <div class="container">

    <div class="col col-content">
      <div class="wrapper">
        <h2 class="has-text-gradient">All your favorite components built-in</h2>
        <p>Air-light comes with minimal styles for typography, navigation and form elements like buttons.</p>
        <p class="button-wrapper">
          <a class="button no-external-link-indicator has-icon" href="https://github.com/digitoimistodude/air-light">
            <svg width="16" height="16" viewBox="0 0 1792 1792" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M1664 896q0 251-146.5 451.5T1139 1625q-27 5-39.5-7t-12.5-30v-211q0-97-52-142 57-6 102.5-18t94-39 81-66.5 53-105T1386 856q0-121-79-206 37-91-8-204-28-9-81 11t-92 44l-38 24q-93-26-192-26t-192 26q-16-11-42.5-27T578 459.5 492 446q-44 113-7 204-79 85-79 206 0 85 20.5 150t52.5 105 80.5 67 94 39 102.5 18q-40 36-49 103-21 10-45 15t-57 5-65.5-21.5T484 1274q-19-32-48.5-52t-49.5-24l-20-3q-21 0-29 4.5t-5 11.5 9 14 13 12l7 5q22 10 43.5 38t31.5 51l10 23q13 38 44 61.5t67 30 69.5 7 55.5-3.5l23-4q0 38 .5 89t.5 54q0 18-13 30t-40 7q-232-77-378.5-277.5T128 896q0-209 103-385.5T510.5 231 896 128t385.5 103T1561 510.5 1664 896z"></path></svg>
            GitHub
          </a>
        </p>
      </div>
    </div>

    <div class="col col-image">
      <div class="wrapper">

          <div class="components">
            <div class="component">
              <span class="component-label">_button-size-large.scss</span>
              <p><a class="button button-large" href="#">Primary button</a></p>
              <p><a class="button button-large focus" href="#">Primary button</a></p>
            </div>

            <div class="component">
              <span class="component-label">_button-size-large.scss</span>
              <p><a class="button button-ghost button-large" href="#">Primary button</a></p>
              <p><a class="button button-ghost button-large focus" href="#">Primary button</a></p>
            </div>

            <div class="component">
              <span class="component-label">_button-style-ghost.scss</span>
              <p><a class="button button-ghost" href="#">Primary button</a></p>
              <p><a class="button button-ghost focus" href="#">Primary button</a></p>
            </div>

            <div class="component">
              <span class="component-label">_button.scss</span>
              <p><a class="button" href="#">Primary button</a></p>
              <p><a class="button focus" href="#">Primary button</a></p>
            </div>

            <div class="component">
              <span class="component-label">_button-size-small.scss</span>
              <p><a class="button button-small" href="#">Tertiary button</a></p>
              <p><a class="button button-small focus" href="#">Tertiary button</a></p>
            </div>
          </div>

      </div>
    </div>

  </div>
<div class="block-name-tag"><p>content-columns</p></div></section>

<section class="block block-content-columns image-right">
  <div class="container">

    <div class="col col-content">
      <div class="wrapper">
        <h2 class="has-text-gradient">Yes, traditional PHP is still a thing</h2>
        <p>We don't use PHP templating languages, because we think everyone should be able to code good-ol' PHP. We keep Air-light consistent with WordPress Coding Standard.</p>
        <p class="button-wrapper">
          <a class="button no-external-link-indicator has-icon" href="https://github.com/digitoimistodude/air-light">
            <svg width="16" height="16" viewBox="0 0 1792 1792" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M1664 896q0 251-146.5 451.5T1139 1625q-27 5-39.5-7t-12.5-30v-211q0-97-52-142 57-6 102.5-18t94-39 81-66.5 53-105T1386 856q0-121-79-206 37-91-8-204-28-9-81 11t-92 44l-38 24q-93-26-192-26t-192 26q-16-11-42.5-27T578 459.5 492 446q-44 113-7 204-79 85-79 206 0 85 20.5 150t52.5 105 80.5 67 94 39 102.5 18q-40 36-49 103-21 10-45 15t-57 5-65.5-21.5T484 1274q-19-32-48.5-52t-49.5-24l-20-3q-21 0-29 4.5t-5 11.5 9 14 13 12l7 5q22 10 43.5 38t31.5 51l10 23q13 38 44 61.5t67 30 69.5 7 55.5-3.5l23-4q0 38 .5 89t.5 54q0 18-13 30t-40 7q-232-77-378.5-277.5T128 896q0-209 103-385.5T510.5 231 896 128t385.5 103T1561 510.5 1664 896z"></path></svg>
            Get started now
          </a>
        </p>

      </div>
    </div>

    <div class="col col-image" aria-hidden="true">
      <div class="wrapper">
        <pre class="code">&lt;<span style="color:#e289bc;">main</span> <span style="color:#7ccb90;">class</span>="<span style="color:#c2b957">site-main</span>"&gt;
  &lt;?php
    the_content<span style="color:#fed600">()</span>;
    air_edit_link<span style="color:#fed600">()</span>;
  ?&gt;
&lt;/<span style="color:#e289bc;">main</span>&gt;</pre>
      </div>
    </div>

  </div>
<div class="block-name-tag" aria-hidden="true"><p>content-columns</p></div>
</section>

<section class="block block-demo-centered">
  <div class="container">
    <h2 class="has-text-gradient">...and more. But still less.</h2>
    <p>Air-light light weights only under 20 KB gzipped and is accessible-ready, gutenberg-ready and multilingual-ready.</p>
    <p class="button-wrapper">
      <a class="button no-external-link-indicator has-icon" href="https://airwptheme.com">
        Check out all the features
      </a>
    </p>
  </div>

<div class="block-name-tag" aria-hidden="true"><p>demo-centered</p></div>
</section>

<!-- Old demo -->
<section class="block has-light-bg" style="display: none;">
  <article class="container article-content">

    <!-- wp:paragraph -->
    <p>You are viewing the demo of <a href="https://www.dude.fi">Digitoimisto Dude Oy's</a> starter theme called <b>Air-light</b> (or air in short). Please go to <a href="https://github.com/digitoimistodude">Dude's GitHub</a> to learn more or <a href="https://github.com/digitoimistodude/air">download the theme</a>.</p>
    <!-- /wp:paragraph -->

    <!-- wp:heading {"level":1} -->
    <h1 class="heading-hero">This is a heading (Hero)</h1>
    <h1>This is a heading (H1)</h1>
    <!-- /wp:heading -->

    <!-- wp:heading {"level":2} -->
    <h2>This is a heading (H2)</h2>
    <!-- /wp:heading -->

    <!-- wp:heading {"level":3} -->
    <h3>This is a heading (H3)</h3>
    <!-- /wp:heading -->

    <!-- wp:heading {"level":4} -->
    <h4>This is a heading (H4)</h4>
    <!-- /wp:heading -->

    <!-- wp:heading {"level":5} -->
    <h5>This is a heading (H5)</h5>
    <!-- /wp:heading -->

    <!-- wp:heading {"level":6} -->
    <h6>This is a heading (H6)</h6>
    <!-- /wp:heading -->

    <!-- wp:heading {"level":2} -->
    <h2 class="heading-demo">Link components</h2>
    <!-- /wp:heading -->

    <div class="components">
      <div class="component">
        <span class="component-label">_link.scss</span>
        <p class="has-align-left"><a href="#" class="link-demo">Link style</a></p>
      </div>
    </div>

    <!-- wp:heading {"level":2} -->
    <h2 class="heading-demo">Paragraph block</h2>
    <!-- /wp:heading -->

    <!-- wp:paragraph -->
    <p>This is a <strong>regular</strong> paragraph block. Professionally productize highly efficient results with world-class core competencies. Objectively matrix leveraged architectures vis-a-vis error-free applications. Completely maximize customized portals via fully researched metrics. Enthusiastically generate premier action items through web-enabled e-markets. Efficiently parallel task holistic intellectual capital and client-centric markets.</p>
    <!-- /wp:paragraph -->

    <!-- wp:heading -->
    <h2 class="heading-demo">Lists in between of text</h2>
    <!-- /wp:heading -->

    <!-- wp:paragraph -->
    <p>This is a <strong>regular</strong> paragraph block. Professionally productize highly efficient results with world-class core competencies. Objectively matrix leveraged architectures vis-a-vis error-free applications. Completely maximize customized portals via fully researched metrics. Enthusiastically generate premier action items through web-enabled e-markets. Efficiently parallel task holistic intellectual capital and client-centric markets.</p>
    <!-- /wp:paragraph -->

    <!-- wp:list -->
    <ul>
    <li>This is ul list item</li>
    <li>This is a long ul list item. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam iaculis fringilla purus, sed feugiat mauris tincidunt ut. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi mauris risus, auctor vel congue et, euismod non mauris.</li>
    <li>This is even longer list item with links in it. <a href="https://www.google.com">External link</a> lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam iaculis fringilla purus, sed feugiat mauris tincidunt ut. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Morbi mauris risus, <a href="https://airwptheme.com/demo">internal link</a> vel congue et, euismod non mauris.</li>
    </ul>
    <!-- /wp:list -->

    <!-- wp:paragraph -->
    <p>This is a <strong>regular</strong> paragraph block. Professionally productize highly efficient results with world-class core competencies. Objectively matrix leveraged architectures vis-a-vis error-free applications. Completely maximize customized portals via fully researched metrics. Enthusiastically generate premier action items through web-enabled e-markets. Efficiently parallel task holistic intellectual capital and client-centric markets.</p>
    <!-- /wp:paragraph -->

    <!-- wp:list {"ordered":true} -->
    <ol>
    <li>This is ol list item</li>
    <li>This is ol list item</li>
    <li>This is ol list item</li>
    </ol>
    <!-- /wp:list -->

    <!-- wp:heading -->
    <h2 class="heading-demo">Traditional items</h2>
    <!-- /wp:heading -->

    <!-- wp:paragraph -->
    <p>This is a <kbd>kbd</kbd> keyboard tag. Like <kbd>alt</kbd> + F4. This is <del>del-tag</del>.</p>
    <!-- /wp:paragraph -->

    <!-- wp:heading -->
    <h2 class="heading-demo">Tables</h2>
    <!-- /wp:heading -->

    <!-- wp:table -->
    <table class="wp-block-table">
      <thead style="display: block; max-width: 315px;">
        <tr>
          <th style="width: 140px;">Name</th>
          <th style="width: 140px;">Age</th>
        </tr>
      </thead>

    <tbody style="display: block; max-width: 315px;">
      <tr>
        <td style="width: 140px;">Antero</td>
        <td style=" width: 140px;">34</td>
      </tr>
      <tr>
        <td style="width: 140px;">Jorma</td>
        <td style="width: 140px;">23</td>
      </tr>
      <tr>
        <td style="width: 140px;">Mikko</td>
        <td style="width: 140px;">19</td>
      </tr>
      <tr>
        <td style="width: 140px;">Päivi</td>
        <td style="width: 140px;">25</td>
      </tr>
      <tr>
        <td style="width: 140px;">Leila</td>
        <td style="width: 140px;">27</td>
      </tr>
      <tr>
        <td style="width: 140px;">Tomi</td>
        <td style="width: 140px;">27</td>
      </tr>
      </tbody>
    </table>
    <!-- /wp:table -->

  </article>
</section>
